styleguide-app/Examples/Menu.elm

```diff
@@ -120,27 +122,30 @@ view state =
                             , ClickableText.custom attrs
                             ]
                 ]
-            }
-            { title = "1st Period English with Mx. Trainer"
-            , icon = viewConfiguration.icon
-            , hasBorder = viewConfiguration.hasBorder
-            , wrapping = viewConfiguration.wrapping
-            , buttonWidth = viewConfiguration.buttonWidth
+            , button =
+                Menu.button
+                    ([ Menu.hasBorder viewConfiguration.hasBorder
+                     , Menu.wrapping viewConfiguration.wrapping
```

Comment:
🎨 How about adding `Menu.emptyAttr` so that we can use existing "generic" maybe functions or casing?
`Maybe.withDefault Menu.emptyAttr viewConfiguration.icon` inside the list of attributes
?

Reply:
oh sorry, I thought that `withOptionalAttribute` was in the API of `Menu`.